.github/workflows/azure-e2e-test.yml

```diff
@@ -116,10 +122,17 @@ jobs:
       with:
         go-version: ${{ env.GO_VERSION }}
 
+    - name: Extract provisioner environment
+      run: |
+        echo "KBS_IMAGE=$(yq -e '.oci.kbs.registry' versions.yaml)" >> "$GITHUB_ENV"
+        echo "KBS_IMAGE_TAG=$(yq -e '.oci.kbs.tag' versions.yaml)" >> "$GITHUB_ENV"
```

Comment:
The current image tag is not compatible with the guest components version right?

Reply:
https://github.com/confidential-containers/cloud-api-adaptor/pull/1789

Reply:
We need to redo this since #1789 was reversed in https://github.com/confidential-containers/cloud-api-adaptor/pull/1800